src/main/java/frc/robot/subsystems/PrimeSwerveModuleSubsystem.java

```diff
@@ -75,20 +73,22 @@ public void setDesiredState(SwerveModuleState desiredState){
    desiredState = SwerveModuleState.optimize(desiredState, encoderRotation);
 
    // Drive motor logic
+      
    var currentVelocityInRotationsPer20ms = RobotMap.kDriveGearRatio * ((mDriveMotor.getSelectedSensorVelocity(0) / 5) / mEncoder.kPositionsPerRotation);
    var currentVelocityInMetersPer20ms = RobotMap.kDriveWheelCircumference * currentVelocityInRotationsPer20ms;
    var desiredVelocity = (desiredState.speedMetersPerSecond / 50) * 2048;
-   var driveFeedforward = mDriveFeedforward.calculate(currentVelocityInMetersPer20ms, desiredVelocity, 0.2);
+   var driveFeedForward = mDriveFeedforward.calculate(getMeasurement(), desiredVelocity, 0.2);
```

Comment:
It looks like you switched the measurement value here for the drive FF calculate call. Does `getMeasurement()` give you the current velocity in rotations per 20ms?